plugins/woocommerce/client/blocks/assets/js/blocks/product-reviews/inner-blocks/reviews-title/style.scss

```
@@ -0,0 +1,4 @@
+
+.wp-block-woocommerce-product-reviews-title.has-background {
```

Should we consider using a class with a WC prefix instead of a WP prefix here? This way, when third-party developers add some styles, they can rely on this specific class for this block?

```suggestion
.wc-block-product-reviews-title.has-background {
```

---

Hmm is this something we do in other places, or do we have some logic that auto adds a `wc-block` generated class?
To be honest, on order for me to do the above I have to add the `wc-block-*` class specifically in multiple places, I rather rely on the auto generated `wp-block` class that is based off of the block name.

---

I see and I somewhat agree with you that it might be better to rely on the auto-generated class instead of specifying the class with the WC prefix in multiple places, as that could lead to more errors. I also asked about this on Slack (p1742547412664939-slack-C02FL3X7KR6).

---

It seems that we have reached a consensus on Slack to use the auto-generated class. Therefore, I am resolving this comment.